src/modules/builder/components/common/ExportRawZipButton.tsx

```diff
@@ -0,0 +1,43 @@
+import { type JSX } from 'react';
+import { useTranslation } from 'react-i18next';
+import { toast } from 'react-toastify';
+
+import { ListItemIcon, MenuItem } from '@mui/material';
+
+import { PackedItem } from '@graasp/sdk';
+
+import { useMutation } from '@tanstack/react-query';
+import { Package } from 'lucide-react';
+
+import { NS } from '@/config/constants';
+import { buildExportAsZipButtonId } from '@/config/selectors';
+import { exportZipMutation } from '@/openapi/client/@tanstack/react-query.gen';
+
+const ExportRawZipButton = ({ item }: { item: PackedItem }): JSX.Element => {
+  const { t: translateBuilder } = useTranslation(NS.Builder);
+  const { t: translateMessage } = useTranslation(NS.Messages);
+
+  const { mutate: exportZip } = useMutation({
+    ...exportZipMutation(),
+    onSuccess: () => {
+      toast.success(translateMessage('EXPORT_RAW_SUCCESS_MESSAGE'));
+    },
+    onError: () => {
+      toast.error(translateMessage('UNEXPECTED_EXPORT_ERROR'));
+    },
+  });
+
+  return (
+    <MenuItem
+      id={buildExportAsZipButtonId(item.id)}
+      onClick={() => exportZip({ path: { itemId: item.id } })}
+    >
+      <ListItemIcon>
+        <Package />
```

Comment:
Nitpick: icons are exported as bare names and with the `Icon` suffix. I prefer using the suffixed version as it makes it clearer that it is an icon. Do as you wish.
```suggestion
        <PackageIcon />
```